base/ipfilter.go

```diff
@@ -0,0 +1,174 @@
+package base
+
+import (
+	"bytes"
+	"github.com/fsnotify/fsnotify"
+	"github.com/spf13/viper"
+	"net"
+	"net/http"
+	"regexp"
+	"strings"
+)
+
+type Prohibited struct {
+	ips     map[string]bool
+	subnets []*subnet
+	ranges  []*iprange
+}
+
+type subnet struct {
+	ip    string
+	ipnet *net.IPNet
+}
+
+type iprange struct {
+	start net.IP
+	end   net.IP
+}
+
+var ProhibitedIPs Prohibited
+var ProhibitedDomains string
+
+func InitProhibitedIPs() {
+	ProhibitedIPs.ips = make(map[string]bool)
+	config := viper.New()
+	config.SetConfigName("prohibited")
+	config.SetConfigType("yaml")
+	config.AddConfigPath("/usr/local/production/config/")
```

Comment:
Can you make the config path an environment variable or external to the source code accessible path ?

Reply:
@vejmarie I have corrected the typo, and changed the path to ENV variable.